formats/json-okio/commonMain/src/kotlinx/serialization/json/OkioStreams.kt

```diff
@@ -0,0 +1,120 @@
+package kotlinx.serialization.json
```

Comment:
This duplicates packages with the regular json library. Isn't this invalid when using Java 9+ modules?

Reply:
Thanks! My bad.